load-testing/tests/relays_stress_test.go

```diff
@@ -336,33 +314,26 @@ func (s *relaysSuite) LocalnetIsRunning() {
 		blockDuration = 60
 	}
 
-	// Set up the blockClient that will be notifying the suite about the committed blocks.
-	s.blockClient = testblock.NewLocalnetClient(s.ctx, s.TestingT.(*testing.T))
-	channel.ForEach(
-		s.ctx,
-		s.blockClient.CommittedBlocksSequence(s.ctx),
-		func(ctx context.Context, block client.Block) {
-			s.latestBlock = block
-		},
-	)
-
 	// Setup the txContext that will be used to send transactions to the network.
 	s.txContext = testtx.NewLocalnetContext(s.TestingT.(*testing.T))
 
 	// Get the relay cost from the tokenomics module.
 	s.relayCoinAmountCost = s.getRelayCost()
 
-	// Setup the tx listener for on-chain events to check and assert on transactions results.
-	s.setupTxEventListeners()
+	// Setup the event listener for on-chain events to check and assert on transactions
```

Comment:
s/on-chain/onchain

https://x.com/olshansky/status/1869115777800180217

Ditto everywhere else (e.g. line 330 below) but I won't leave any more comments.

Please do a find-all "on-chain"

Reply:
Replaced all the `on-chain/off-chain` occurrences with `onchain/offchain` but seemed too much changes for this already big PR.

Addressed them #1020